### ISSUE TITLE
Bug: A space is required after ','. (comma-spacing)

### ISSUE THREAD

**Issue by bucaran** [eslint/eslint#2618]
```js
util.log(`${pkg.name}, version ${pkg.version}`)
```

I just checked again, and there was indeed a space after the comma.

Ideas?

This is my `.eslintrc`
```json
{
  "parser" : "babel-eslint",
  "env" : {
    "node" : true,
    "es6" : true
  },
  "rules" : {
    "semi" : false,
    "curly": false,
    "no-use-before-define" : false,
    "strict" : false ,
    "no-var" : 2,
    "no-unused-expressions" : false,
    "no-shadow" : false,
    "no-underscore-dangle" : false,
    "space-before-function-paren" : "always"
  }
}
```

Thanks!

**Comment by gyandeeps** [comment]
Can you share:

* version of eslint being used?
* What kind of parser?
* Console output of eslint

**Comment by xjamundx** [comment]
FYI no errors with raw ESLint (not babel-eslint) version `0.21.0`:

> eslint --rule {strict:0} --env es6 test.js

```
var pkg = {};
var str = `${pkg.name}, version ${pkg.version}`;
console.log(str);
```

**Comment by hzoo** [comment]
What version of `babel-eslint`? I should of fixed this in `3.1.9`. Is it just that single line giving an error because I can't reproduce it. Also if it's specific to `babel-eslint` then report it in that repo :smile:

**Comment by xjamundx** [comment]
Pretty sure we can close this one here!

**Comment by bucaran** [comment]
Updated to 0.21.0 and still getting the same problem.

**Comment by bucaran** [comment]
Alright, upgraded to `babel-eslint 3.1.9` and it's fixed. One thing though, now it does not show the error if I do it wrong, ie. not adding the space after the comma.

:sigh:

**Comment by hzoo** [comment]
I think that's right because it's a comma in a template string not a comma in a parameter list. It would be the same as "a,b" - the rule wouldn't error for the comma inside of a regular string right?

**Comment by lo1tuma** [comment]
Yes, that is the expected behavior, comma within a template string should be ignored by this rule.

**Comment by ilyavolodin** [comment]
Closing as this is behaving correctly.

**Comment by bucaran** [comment]
@lo1tuma Well, yes, unless you set this rule:

```js
{ "space-before-function-paren" : "always" }
```

Correct? So, is still behaving incorrectly.
